assets/images/Svg.js

```diff
@@ -29,7 +29,8 @@ const svg = {
   "whole-word-match": require("./whole-word-match.svg"),
   "worker": require("./worker.svg"),
   "sad-face": require("./sad-face.svg"),
-  "refresh": require("./refresh.svg")
+  "refresh": require("./refresh.svg"),
+  "fastForward": require("./fast-forward.svg")
```

Comment:
Not sure about the naming convention for this newly added svg file. It is used as a dropdown for hidden tabs, but the symbol represents fast-forward.